### DISCUSSION TITLE
Welcome to Euler Discussions!

### DISCUSSION THREAD

**Discussion by arguiot** [arguiot/Euler#13] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Caffe-del** [comment]
Hi,  Today I want to use this Euler to calculate an express. Such as 
`999 * 999`, And I want to get the result is '998,001'. 
Can I use the Euler api to finish it?

**Reply by arguiot** [reply]
I'm not sure what you mean. Do you have "999 * 999" as a string? If so you can use `Parser`. Otherwise if you have this in code, just use `BN(999)` to convert to Euler notation and you can just do `a * b`.
